examples/openai/pyfunc.py

```diff
@@ -13,6 +13,14 @@
         messages=[{"role": "system", "content": "You are an MLflow expert!"}],
     )
 
+
```

Comment:
Can we add setting token scope here for completeness?
```
# On Databricks, set the stored OpenAI API key scope here for automatically loading the API key for real time inference.
# See https://docs.databricks.com/security/secrets/index.html on how to add a scope and API key.
os.environ["MLFLOW_OPENAI_SECRET_SCOPE"] = "<scope-name>"
```

Reply:
Sure!

Reply:
done